.ostree/packages-testing-RedHat-10.txt

```diff
@@ -0,0 +1,3 @@
+bash
+man-db
+openssh-keycat
```

Comment:
given these files are same (and I think most of the others here), would it make sense to symlink them similarly as we do with the var files?

Reply:
done